server/service/mdm.go

```diff
@@ -887,6 +887,46 @@ func (svc *Service) GetMDMDiskEncryptionSummary(ctx context.Context, teamID *uin
 	return nil, fleet.ErrMissingLicense
 }
 
+////////////////////////////////////////////////////////////////////////////////
+// GET /mdm/profiles/summary
+////////////////////////////////////////////////////////////////////////////////
+
+type getMDMProfilesSummaryRequest struct {
+	TeamID *uint `query:"team_id,optional"`
+}
+
+type getMDMProfilesSummaryResponse struct {
+	fleet.MDMProfilesSummary
+	Err error `json:"error,omitempty"`
+}
+
+func (r getMDMProfilesSummaryResponse) error() error { return r.Err }
+
+func getMDMProfilesSummaryEndpoint(ctx context.Context, request interface{}, svc fleet.Service) (errorer, error) {
+	req := request.(*getMDMProfilesSummaryRequest)
+	res := getMDMProfilesSummaryResponse{}
+
+	// TODO: Think about how to handle authz for this endpoint. As it is, we are
+	// authorizing the user once for each platform because of the legacy Apple endpoint.
```

Comment:
I think this is fine, the authorization is the same for Apple and Windows (`mdm_config_profile`, platform-agnostic). So if they are not authorized, it will fail on the first call, and if they are, it will succeed on the second too.